main.js

```diff
@@ -844,28 +844,48 @@ require('es6-promise').polyfill();
     }
   }
 
-  proto.requestMessagesSinceTimestamp = function requestMessagesSinceTimestamp(lastTimestamp, quantity, withGroups){
-    // if (this.socketConnection == null) {
-    //
-    // }
+  proto.unsend = function unsend(message){
```

Comment:
I need messageId, senderId and recipientId, i think it's easier for the user to pass just the message instead of remembering to pass 3 params